handlebars/vue/packages/vue-app/src/main.js.hbs

```diff
@@ -14,7 +14,7 @@ const cache = new InMemoryCache();
 // HTTP connection to the API
 const httpLink = createHttpLink({
   // You should use an absolute URL here
-  uri: "https://api.thegraph.com/subgraphs/name/aave/protocol",
+  uri: "{{ subgraphUri }}",
```

Comment:
```suggestion
  uri: "{{ subgraphUrl }}",
```
There's no definition for `subgraphUrl` so the Vue app ends up querying localhost (and obviously failing)